src/components/books.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import AddBook from './addbook';
+
+export default function Books() {
+  return (
+    <div id="main-container">
+      <div id="books-container" />
+      <AddBook />
```

Comment:
- [ ] This section should take the addbook as props to render the author and title